Source/WebCore/platform/mock/mediasource/MockMediaPlayerMediaSource.cpp

```diff
@@ -121,10 +121,8 @@ void MockMediaPlayerMediaSource::cancelLoad()
 void MockMediaPlayerMediaSource::play()
 {
     m_playing = 1;
-    callOnMainThread([this, weakThis = WeakPtr { *this }] {
-        if (!weakThis)
-            return;
-        advanceCurrentTime();
+    callOnMainThread([protectedThis = Ref { *this }] {
```

Comment:
Why are we doing this change and not refing in the callback?
Ditto below

Reply:
we want to keep a refcount on the player , that was Jer original intent